### DISCUSSION TITLE
CMI /silence does not hide join/quit messages

### DISCUSSION THREAD

**Discussion by ChiboYen** [Zrips/CMI#5070] [Enhancement]
**Description of issue:**
/silence does not hide join/quit messages
![image](https://user-images.githubusercontent.com/69435677/107114477-4672ba00-6823-11eb-8865-43f275ac2b2b.png)

---

**Cmi Version (using`/cmi version`):** 8.8.1.0

**Server Type (Spigot/Paperspigot/etc):** 1.16.4 Paper

**Server Version (using `/ver`):** paper 462

**Comment by Zrips** [comment]
As stated in commands description, it in general only hides public messages as messages like join/leave requires quite a different approach to be hidden from player as those are not sent by plugin itself but by server.

**Comment by ChiboYen** [comment]
Alright, if you could consider adding this feature to the silence command, I would really appreciate it. Players wish to hide chat but they keep getting join/leave message spams.
